File: Obnovljivac.API/ClientApp/src/app/models/autogenerated/times.ts

// ----------------------------------------------
//  Interface and model
// ----------------------------------------------
export interface ITimes  {
    data: number;
    process: number;
}

export class Times  implements ITimes {
    public data: number;
    public process: number;

    constructor(data: number = null,  process: number = null ) {
        
        this.data = data;
        this.process = process;
    }
}

 

File: Obnovljivac.API/Models/NasaPower/Times.cs
namespace Obnovljivac.API.Models.NasaPower
{
    /// <summary>
    /// Model from 
    /// <see href="https://power.larc.nasa.gov/api/pages/?urls.primaryName=Daily">
    /// POWER Daily API
    /// </see>
    /// <br></br>
    /// The API allows daily data requests of POWER Analysis Ready Data (ARD).
    /// </summary>
    public class Times
    {
        public double Data { get; set; }
        public double Process { get; set; }
    }
}
